torchtitan/experiments/README.md

```diff
@@ -0,0 +1,18 @@
+## Contributing Guidelines
+
```

Comment:
Maybe a friendlier start - ala "To help users contribute and improve 'torchtitan' in a more rapid fashion, we are adding a new, experimental section.  The general contribution guidelines for this are below and we look forward to your innovations! "